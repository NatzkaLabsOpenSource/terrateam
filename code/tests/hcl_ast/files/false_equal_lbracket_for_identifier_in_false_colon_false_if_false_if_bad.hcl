foo = [for foo in foo : foo if foo :
---
foo = [for foo in foo : foo if foo if
