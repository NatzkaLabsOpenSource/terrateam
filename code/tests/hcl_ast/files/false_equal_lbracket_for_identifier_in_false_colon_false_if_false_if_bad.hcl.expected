Unexpected operator in "if" clause of "for".
